tests/ui/swap.rs

```diff
@@ -5,7 +5,7 @@
     clippy::blacklisted_name,
     clippy::no_effect,
     clippy::redundant_clone,
-    redundant_semicolons,
+    redundant_semicolon,
```

Comment:
This is not necessary. Everything else LGTM

```suggestion
    redundant_semicolons,
```